Cannot build on Windows.(undefined reference to ...)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

I have been trying to build the latest source with MSYS2 UCRT64 but failed.

There are tons of logs for 'undefined reference to...'.

logs are below:


transforming (102) ..\..\..\..\node_modules\@vue\devtools-api\lib\esm\env.jsC:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.
c.obj):(.text+0xab739): undefined reference to `__imp_gnutls_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0xab921): undefined reference to `__imp_gnutls_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0x171): undefined reference to `libintl_vasprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0x542): undefined reference to `libintl_vasprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0x5a): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0xcb): undefined reference to `libintl_fprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(kx.o):(.text+0x5f1): undefined reference to `libintl_fprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x16dd): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x177a): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x17a3): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x17de): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x1801): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x1824): more undefined references to `libintl_dgettext' follow
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0xca): undefined reference to `libintl_bindtextdomain'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x198): undefined reference to `asn1_check_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x1c2): undefined reference to `asn1_array2tree'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x240): undefined reference to `asn1_check_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x2b2): undefined reference to `asn1_array2tree'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x414): undefined reference to `asn1_check_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x599): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_6  vite:resolve 87.12ms ../webfonts/fa-brands-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2 +0ms
4-w64  vite:resolve 87.25ms ../webfonts/fa-regular-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2 +0ms
-mingw32/bin/ld.exe  vite:resolve 87.40ms ../webfonts/fa-solid-900.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 +0ms
:   vite:resolve 87.79ms ../webfonts/fa-brands-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2 +0ms
C:/msys64/ucrt64/bin/../lib/gcc/x8  vite:resolve 88.04ms ../webfonts/fa-solid-900.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 +0ms
6_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o)  vite:resolve 88.56ms ../webfonts/fa-regular-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2 +0ms
:(.text  vite:resolve 89.51ms ../webfonts/fa-solid-900.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 +0ms
+0x  vite:resolve 89.68ms ../webfonts/fa-brands-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2 +0ms
5a5  vite:resolve 89.82ms ../webfonts/fa-regular-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2 +0ms
): undefined reference to `  vite:resolve 89.94ms ../webfonts/fa-v4compatibility.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-v4compatibility.woff2 +0ms
asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x51d): un  vite:resolve 0.43ms ../webfonts/fa-regular-400.ttf -> C
:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.ttf +4ms
defined reference to `libintl_fprintf  vite:resolve 0.34ms ../webfonts/fa-brands-400.ttf -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.ttf +1ms
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe  vite:resolve 0.30ms ../webfonts/fa-solid-900.ttf -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.ttf +1ms
: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-  vite:resolve 0.37ms ../webfonts/fa-v4compatibility.ttf -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-v4compatibility.ttf +1ms
w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text.startup+0x81): undefined reference to `libintl_fprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(alert.o):(.text+0xcd): undefined reference to `libintl_dgettext'
-mingw32/14.1.0/../../../../lib\libgnutls.a(alert.o)transforming (137) ..\..\..\..\node_modules\@fortawesome\fontawesome-free\css\all.min.css:(.text+0x29): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(str.o):(.text+0x78e): undefined reference to `libintl_vasprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(str.o):(.text+0xb0d): undefined reference to `libintl_sprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(str.o):(.text+0xb66): undefined reference to `libintl_sprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x8fd): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x96e): undefined reference to `asn1_der_decoding'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x999): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x9cd): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xa4b): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xb87): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xcf7): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xd95): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xed2): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xf70): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x1d9e): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x1dcb): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x1e25): undefined reference to `asn1_der_decoding2'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2cc4): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2e1f): undefined reference to `asn1_der_decoding'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2e57): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2f5d): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2ff4): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x307b): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x3149): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x3278): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x32c9): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x336b): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x33b4): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x3a): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x65): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x9d): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x19e): undefined reference to `p11_kit_uri_match_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x1ad): undefined reference to `p11_kit_uri_match_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x1b9): undefined reference to `p11_kit_module_get_flags'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x27b): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2e7): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x41e): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x444): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x4b1): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x599): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x5d9): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x6c5): undefined reference to `p11_kit_uri_match_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x6d4): undefined reference to `p11_kit_uri_match_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x6e7): undefined reference to `p11_kit_config_option'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x8bf): undefined reference to `p11_kit_module_get_flags'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x9dc): undefined reference to `p11_kit_modules_load_and_initialize
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xa15): undefined reference to `p11_kit_module_get_name'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xb01): undefined reference to `p11_kit_message'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xca5): undefined reference to `p11_kit_space_strdup'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xcb6): undefined reference to `p11_kit_uri_new'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xcca): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xd40): undefined reference to `p11_kit_uri_format'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xd50): undefined reference to `p11_kit_uri_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xdd3): undefined reference to `p11_kit_pin_new_for_string'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xeac): undefined reference to `p11_kit_uri_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x119c): undefined reference to `p11_kit_uri_match_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x11b3): undefined reference to `p11_kit_uri_match_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x214c): undefined reference to `p11_kit_module_load'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x21b3): undefined reference to `p11_kit_module_release'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x21f4): undefined reference to `p11_kit_module_release'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2234): undefined reference to `p11_kit_module_initialize'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2244): undefined reference to `p11_kit_module_finalize'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x264f): undefined reference to `p11_kit_pin_register_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2781): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2801): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2829): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x284a): undefined reference to `p11_kit_space_strlen'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2861): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2871): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2881): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2891): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x28a6): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x28d6): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2926): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2a0f): undefined reference to `p11_kit_pin_register_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2ae8): undefined reference to `p11_kit_module_release'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2b09): undefined reference to `p11_kit_module_finalize'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2bb2): undefined reference to `p11_kit_uri_parse'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2c2b): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2c3e): undefined reference to `p11_kit_uri_new'


### Expected Behavior

_No response_

### Additional Context

_No response_

### Host Operating System

Windows

### Operating System Version

WIN 10 22H2

### Architecture

64 bit

### Sunshine commit or version

The latest

### Package

Windows - portable

### GPU Type

Nvidia

### GPU Model

RTX 4060

### GPU Driver/Mesa Version

None

### Capture Method

None

### Config

```Shell
-
```


### Apps

_No response_

### Relevant log output

```shell
transforming (102) ..\..\..\..\node_modules\@vue\devtools-api\lib\esm\env.jsC:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.
c.obj):(.text+0xab739): undefined reference to `__imp_gnutls_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(unity_0_c.c.obj):(.text+0xab921): undefined reference to `__imp_gnutls_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0x171): undefined reference to `libintl_vasprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0x542): undefined reference to `libintl_vasprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0x5a): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(errors.o):(.text+0xcb): undefined reference to `libintl_fprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(kx.o):(.text+0x5f1): undefined reference to `libintl_fprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x16dd): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x177a): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x17a3): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x17de): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x1801): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(cert-cred.o):(.text+0x1824): more undefined references to `libintl_dgettext' follow
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0xca): undefined reference to `libintl_bindtextdomain'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x198): undefined reference to `asn1_check_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x1c2): undefined reference to `asn1_array2tree'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x240): undefined reference to `asn1_check_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x2b2): undefined reference to `asn1_array2tree'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x414): undefined reference to `asn1_check_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x599): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_6  vite:resolve 87.12ms ../webfonts/fa-brands-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2 +0ms
4-w64  vite:resolve 87.25ms ../webfonts/fa-regular-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2 +0ms
-mingw32/bin/ld.exe  vite:resolve 87.40ms ../webfonts/fa-solid-900.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 +0ms
:   vite:resolve 87.79ms ../webfonts/fa-brands-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2 +0ms
C:/msys64/ucrt64/bin/../lib/gcc/x8  vite:resolve 88.04ms ../webfonts/fa-solid-900.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 +0ms
6_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o)  vite:resolve 88.56ms ../webfonts/fa-regular-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2 +0ms
:(.text  vite:resolve 89.51ms ../webfonts/fa-solid-900.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 +0ms
+0x  vite:resolve 89.68ms ../webfonts/fa-brands-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2 +0ms
5a5  vite:resolve 89.82ms ../webfonts/fa-regular-400.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2 +0ms
): undefined reference to `  vite:resolve 89.94ms ../webfonts/fa-v4compatibility.woff2 -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-v4compatibility.woff2 +0ms
asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text+0x51d): un  vite:resolve 0.43ms ../webfonts/fa-regular-400.ttf -> C
:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.ttf +4ms
defined reference to `libintl_fprintf  vite:resolve 0.34ms ../webfonts/fa-brands-400.ttf -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.ttf +1ms
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe  vite:resolve 0.30ms ../webfonts/fa-solid-900.ttf -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.ttf +1ms
: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-  vite:resolve 0.37ms ../webfonts/fa-v4compatibility.ttf -> C:/msys64/home/user/sunshine_orig/node_modules/@fortawesome/fontawesome-free/webfonts/fa-v4compatibility.ttf +1ms
w64-mingw32/14.1.0/../../../../lib\libgnutls.a(global.o):(.text.startup+0x81): undefined reference to `libintl_fprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(alert.o):(.text+0xcd): undefined reference to `libintl_dgettext'
-mingw32/14.1.0/../../../../lib\libgnutls.a(alert.o)transforming (137) ..\..\..\..\node_modules\@fortawesome\fontawesome-free\css\all.min.css:(.text+0x29): undefined reference to `libintl_dgettext'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(str.o):(.text+0x78e): undefined reference to `libintl_vasprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(str.o):(.text+0xb0d): undefined reference to `libintl_sprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(str.o):(.text+0xb66): undefined reference to `libintl_sprintf'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x8fd): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x96e): undefined reference to `asn1_der_decoding'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x999): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x9cd): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xa4b): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xb87): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xcf7): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xd95): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xed2): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0xf70): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x1d9e): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x1dcb): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x1e25): undefined reference to `asn1_der_decoding2'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2cc4): undefined reference to `asn1_create_element'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2e1f): undefined reference to `asn1_der_decoding'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2e57): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2f5d): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x2ff4): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x307b): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x3149): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x3278): undefined reference to `asn1_read_value'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x32c9): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x336b): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pubkey.o):(.text+0x33b4): undefined reference to `asn1_delete_structure'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x3a): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x65): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x9d): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x19e): undefined reference to `p11_kit_uri_match_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x1ad): undefined reference to `p11_kit_uri_match_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x1b9): undefined reference to `p11_kit_module_get_flags'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x27b): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2e7): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x41e): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x444): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x4b1): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x599): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x5d9): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x6c5): undefined reference to `p11_kit_uri_match_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x6d4): undefined reference to `p11_kit_uri_match_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x6e7): undefined reference to `p11_kit_config_option'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x8bf): undefined reference to `p11_kit_module_get_flags'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x9dc): undefined reference to `p11_kit_modules_load_and_initialize
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xa15): undefined reference to `p11_kit_module_get_name'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xb01): undefined reference to `p11_kit_message'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xca5): undefined reference to `p11_kit_space_strdup'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xcb6): undefined reference to `p11_kit_uri_new'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xcca): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xd40): undefined reference to `p11_kit_uri_format'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xd50): undefined reference to `p11_kit_uri_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xdd3): undefined reference to `p11_kit_pin_new_for_string'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0xeac): undefined reference to `p11_kit_uri_free'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x119c): undefined reference to `p11_kit_uri_match_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x11b3): undefined reference to `p11_kit_uri_match_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x214c): undefined reference to `p11_kit_module_load'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x21b3): undefined reference to `p11_kit_module_release'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x21f4): undefined reference to `p11_kit_module_release'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2234): undefined reference to `p11_kit_module_initialize'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2244): undefined reference to `p11_kit_module_finalize'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x264f): undefined reference to `p11_kit_pin_register_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2781): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2801): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2829): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x284a): undefined reference to `p11_kit_space_strlen'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2861): undefined reference to `p11_kit_uri_get_module_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2871): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2881): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2891): undefined reference to `p11_kit_uri_get_token_info'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x28a6): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x28d6): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2926): undefined reference to `p11_kit_uri_get_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2a0f): undefined reference to `p11_kit_pin_register_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2ae8): undefined reference to `p11_kit_module_release'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2b09): undefined reference to `p11_kit_module_finalize'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2bb2): undefined reference to `p11_kit_uri_parse'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2c2b): undefined reference to `p11_kit_uri_set_attribute'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libgnutls.a(pkcs11.o):(.text+0x2c3e): undefined reference to `p11_kit_uri_new'
```

> Here's how you can do it:
> 
> https://github.com/LizardByte/Sunshine/blob/722e5600c6a15a605b5f07e17a17045fb19290cd/.github/workflows/CI.yml#L991-L995

Errors are different but still exists.

C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x13c): undefined reference to `nghttp2_session_check_reque
st_allowed'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2a4): undefined reference to `nghttp2_session_mem_recv'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x324): undefined reference to `nghttp2_session_check_reque
st_allowed'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x368): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x594): undefined reference to `nghttp2_session_want_read'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x5a4): undefined reference to `nghttp2_session_want_write'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x6ed): undefined reference to `nghttp2_priority_spec_init'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x7eb): undefined reference to `nghttp2_submit_priority'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x7f8): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x85d): undefined reference to `nghttp2_session_want_write'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x869): undefined reference to `nghttp2_session_send'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x876): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x8b5): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x9a9): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0xad2): undefined reference to `nghttp2_session_set_stream_
user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0xb43): undefined reference to `nghttp2_submit_rst_stream'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0xb4b): undefined reference to `nghttp2_session_send'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x10db): undefined reference to `nghttp2_http2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1113): undefined reference to `nghttp2_http2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1221): undefined reference to `nghttp2_session_get_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x12e9): undefined reference to `nghttp2_session_get_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1397): undefined reference to `nghttp2_session_set_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1424): undefined reference to `nghttp2_http2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x14a1): undefined reference to `nghttp2_session_set_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1535): undefined reference to `nghttp2_session_get_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x17ca): undefined reference to `nghttp2_session_want_write
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x184a): undefined reference to `nghttp2_session_get_remote
_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x189a): undefined reference to `nghttp2_session_get_stream
_remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1917): undefined reference to `nghttp2_session_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1b9e): undefined reference to `nghttp2_session_get_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x1eea): undefined reference to `nghttp2_submit_rst_stream'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x25bf): undefined reference to `nghttp2_session_get_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x263c): undefined reference to `nghttp2_submit_rst_stream'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x266c): undefined reference to `nghttp2_session_consume'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x26e9): undefined reference to `nghttp2_session_consume'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/..[ 91%] /lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o)Building CXX object tests/CMakeFiles/test_sunshine.dir/__/th
ird-party/ViGEmClient/src/ViGEmClient.cpp.obj:
(.text+0x2b14): undefined reference to `nghttp2_session_get_local_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2b26): undefined reference to `nghttp2_session_get_stream
_effective_local_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2b38): undefined reference to `nghttp2_session_get_stream
_effective_recv_data_length'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2c71): undefined reference to `nghttp2_session_set_local_
window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2d58): undefined reference to `nghttp2_session_resume_dat
a'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2d9a): undefined reference to `nghttp2_session_set_local_
window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2df0): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2e4a): undefined reference to `nghttp2_submit_ping'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2e59): undefined reference to `nghttp2_session_send'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2e83): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x2eab): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3119): undefined reference to `nghttp2_session_callbacks_
new'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3137): undefined reference to `nghttp2_session_callbacks_
set_send_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3148): undefined reference to `nghttp2_session_callbacks_
set_on_frame_recv_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3159): undefined reference to `nghttp2_session_callbacks_
set_on_frame_send_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x316a): undefined reference to `nghttp2_session_callbacks_
set_on_data_chunk_recv_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x317b): undefined reference to `nghttp2_session_callbacks_
set_on_stream_close_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x318c): undefined reference to `nghttp2_session_callbacks_
set_on_begin_headers_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x319d): undefined reference to `nghttp2_session_callbacks_
set_on_header_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/..[ 92%] /lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o)Building CXX object tests/CMakeFiles/test_sunshine.dir/__/sr
c/platform/windows/nvprefs/driver_settings.cpp.obj:
(.text+0x31ae): undefined reference to `nghttp2_session_callbacks_set_error_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x31bf): undefined reference to `nghttp2_option_new'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x31eb): undefined reference to `nghttp2_option_set_no_auto
_window_update'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x31fa): undefined reference to `nghttp2_option_set_no_rfc9
113_leading_and_trailing_ws_validation'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x320d): undefined reference to `nghttp2_session_client_new
2'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x321a): undefined reference to `nghttp2_option_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3297): undefined reference to `nghttp2_pack_settings_payl
oad'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x32cb): undefined reference to `nghttp2_session_callbacks_
del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x334a): undefined reference to `nghttp2_submit_settings'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3367): undefined reference to `nghttp2_session_set_local_
window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3421): undefined reference to `nghttp2_session_upgrade2'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x343d): undefined reference to `nghttp2_session_set_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x34f3): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3528): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3553): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x398c): undefined reference to `nghttp2_session_get_stream
_remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3a23): undefined reference to `nghttp2_session_get_remote
_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3a37): undefined reference to `nghttp2_session_get_stream
_remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3c9b): undefined reference to `nghttp2_session_get_remote
_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o)[ 92%] :Building CXX object tests/CMakeFiles/test_sunshine.dir/__/s
rc/platform/windows/nvprefs/nvapi_opensource_wrapper.cpp.obj(
.text+0x3d04): undefined reference to `nghttp2_session_get_stream_remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3e6d): undefined reference to `nghttp2_session_want_read'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3e7e): undefined reference to `nghttp2_session_want_write
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3f6c): undefined reference to `nghttp2_session_resume_dat
a'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3f73): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x3fb5): undefined reference to `nghttp2_session_want_read'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe[ 93%] : Building CXX object tests/CMakeFiles/test_sunshine.dir/__/src/platform/windows/nvprefs/nvprefs_common.cpp.objC:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.
1.0/../../../../lib\libcurl.a(libcurl_la-http2.o)
:(.text+0x3fc6): undefined reference to `nghttp2_session_want_write'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4091): undefined reference to `nghttp2_session_check_requ
est_allowed[ 94%] '
Building CXX object tests/CMakeFiles/test_sunshine.dir/__/src/platform/windows/nvprefs/nvprefs_interface.cpp.objC:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe
: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x416d): undefined reference to `nghttp2_submit_request'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x43d2): undefined reference to `nghttp2_session_get_remote
_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x44ac): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x479f): undefined reference to `nghttp2_version'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4997): undefined reference to `nghttp2_session_get_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4a01): undefined reference to `nghttp2_session_get_remote
_settings'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4a14): undefined reference to `nghttp2_session_get_remote
_settings'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4d13): undefined reference to `nghttp2_session_get_stream
_effective_local_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4d26): undefined reference to `nghttp2_session_get_stream
_effective_recv_data_length'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o)[ 95%] :Building CXX object tests/CMakeFiles/test_sunshine.dir/__/s
rc/platform/windows/nvprefs/undo_data.cpp.obj(
.text+0x4ee0): undefined reference to `nghttp2_submit_rst_stream'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x4ee9): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x5187): undefined reference to `nghttp2_submit_rst_stream'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x518e): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x54ad): undefined reference to `nghttp2_session_set_stream
_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-http2.o):(.text+0x5587): undefined reference to `nghttp2_pack_settings_payl
oad'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-openssl.o):(.text+0x526): undefined reference to `SSL_alert_desc_string_lon
g'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-openssl.o):(.text+0x65bc): undefined reference to `OCSP_basic_verify'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-openssl.o):(.text+0x691f): undefined reference to `OCSP_response_status_str
'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-openssl.o):(.text+0x69e2): undefined reference to `OCSP_cert_status_str'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-openssl.o):(.text+0x6a1d): undefined reference to `OCSP_crl_reason_str'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0xd4): undefined reference to `nghttp2_session_mem_re
cv'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1ec): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x504): undefined reference to `nghttp2_http2_strerro
r'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x58d): undefined reference to `nghttp2_session_want_
write'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x599): undefined reference to `nghttp2_session_send'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x5a6): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x5f5): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x639): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0xa94): undefined reference to `nghttp2_session_consu
me'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0xb82): undefined reference to `nghttp2_session_consu
me'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0xea3): undefined reference to `nghttp2_http2_strerro
r'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1043): undefined reference to `nghttp2_session_resu
me_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x104a): undefined reference to `nghttp2_is_fatal'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x109b): undefined reference to `nghttp2_session_get_
stream_remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x10e9): undefined reference to `nghttp2_session_get_
remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x127b): undefined reference to `nghttp2_session_get_
remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x128c): undefined reference to `nghttp2_session_get_
stream_remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1314): undefined reference to `nghttp2_session_want
_read'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1404): undefined reference to `nghttp2_session_want
_write'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1585): undefined reference to `nghttp2_session_get_
remote_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1619): undefined reference to `nghttp2_session_get_
st[ 96%] ream_remote_window_sizeBuilding CXX object tests/CMakeFiles/test_sunshine.dir/__/src/platform/windows/nvprefs/undo_file.cpp.obj'

C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x163c): undefined referenc[e 96%]  to `Building C ob
ject tests/CMakeFiles/test_sunshine.dir/__/third-party/tray/src/tray_windows.c.objn
ghttp2_session_want_write'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x16e7): undefined reference to `nghttp2_session_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x1df7): undefined reference to `nghttp2_session_get_
stream_user_data'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2160): undefined reference to `nghttp2_submit_reque
st'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x23f4): undefined reference to `nghttp2_session_call
backs_new'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2414): undefined reference to `nghttp2_session_call
backs_set_send_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2425): undefined reference to `nghttp2_session_call
backs_set_on_frame_recv_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2436): undefined reference to `nghttp2_session_call
backs_set_on_frame_send_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2447): undefined reference to `nghttp2_session_call
backs_set_on_data_chunk_recv_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2458): undefined reference to `nghttp2_session_call
backs_set_on_stream_close_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2469): undefined reference to `nghttp2_session_call
backs_set_on_header_callback'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2481): undefined reference to `nghttp2_option_new'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x24a7): undefined reference to `nghttp2_session_call
backs_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x26dd): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x29db): undefined reference to `nghttp2_option_set_n
o_auto_window_update'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x29ea): undefined reference to `nghttp2_option_set_n
o_rfc9113_leading_and_trailing_ws_validation'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x29ff): undefined reference to `nghttp2_session_clie
nt_new2'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2a0b): undefined reference to `nghttp2_option_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2a67): undefined reference to `nghttp2_submit_setti
ngs'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2a84): undefined reference to `nghttp2_session_set_
local_window_size'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2a9d): undefined reference to `nghttp2_session_call
backs_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2b2b): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2b4f): undefined reference to `nghttp2_session_call
backs_del'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2b99): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x2c27): undefined reference to `nghttp2_strerror'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x34e3): undefined reference to `nghttp2_session_want
_read'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../lib\libcurl.a(libcurl_la-cf-h2-proxy.o):(.text+0x34f3): undefined reference to `nghttp2_session_want
_write'

--- a/.codeql-prebuild-cpp-Windows.sh
+++ b/.codeql-prebuild-cpp-Windows.sh
@@ -2,18 +2,10 @@
 set -e
 
 # update pacman
-pacman --noconfirm -Suy
-
-# install wget
-pacman --noconfirm -S \
-  wget
-
-# download working curl
-wget https://repo.msys2.org/mingw/ucrt64/mingw-w64-ucrt-x86_64-curl-8.8.0-1-any.pkg.tar.zst
+pacman --noconfirm -Syu
 
 # install dependencies
-pacman -U --noconfirm mingw-w64-ucrt-x86_64-curl-8.8.0-1-any.pkg.tar.zst
-pacman -Syu --noconfirm --ignore=mingw-w64-ucrt-x86_64-curl \
+pacman -S --noconfirm \
   base-devel \
   diffutils \
   gcc \
@@ -21,6 +13,7 @@ pacman -Syu --noconfirm --ignore=mingw-w64-ucrt-x86_64-curl \
   make \
   mingw-w64-ucrt-x86_64-cmake \
   mingw-w64-ucrt-x86_64-cppwinrt \
+  mingw-w64-ucrt-x86_64-curl-winssl \
   mingw-w64-ucrt-x86_64-graphviz \
   mingw-w64-ucrt-x86_64-miniupnpc \
   mingw-w64-ucrt-x86_64-nlohmann-json \

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -1019,30 +1019,20 @@ jobs:
           msystem: ucrt64
           update: true
           install: >-
-            wget
-
-      - name: Update Windows dependencies
-        shell: msys2 {0}
-        run: |
-          # download working curl
-          wget https://repo.msys2.org/mingw/ucrt64/mingw-w64-ucrt-x86_64-curl-8.8.0-1-any.pkg.tar.zst
-
-          # install dependencies
-          pacman -U --noconfirm mingw-w64-ucrt-x86_64-curl-8.8.0-1-any.pkg.tar.zst
-          pacman -Syu --noconfirm \
-            --ignore=mingw-w64-ucrt-x86_64-curl \
-            git \
-            mingw-w64-ucrt-x86_64-cmake \
-            mingw-w64-ucrt-x86_64-cppwinrt \
-            mingw-w64-ucrt-x86_64-graphviz \
-            mingw-w64-ucrt-x86_64-miniupnpc \
-            mingw-w64-ucrt-x86_64-nlohmann-json \
-            mingw-w64-ucrt-x86_64-nodejs \
-            mingw-w64-ucrt-x86_64-nsis \
-            mingw-w64-ucrt-x86_64-onevpl \
-            mingw-w64-ucrt-x86_64-openssl \
-            mingw-w64-ucrt-x86_64-opus \
+            git
+            mingw-w64-ucrt-x86_64-cmake
+            mingw-w64-ucrt-x86_64-cppwinrt
+            mingw-w64-ucrt-x86_64-curl-winssl
+            mingw-w64-ucrt-x86_64-graphviz
+            mingw-w64-ucrt-x86_64-miniupnpc
+            mingw-w64-ucrt-x86_64-nlohmann-json
+            mingw-w64-ucrt-x86_64-nodejs
+            mingw-w64-ucrt-x86_64-nsis
+            mingw-w64-ucrt-x86_64-onevpl
+            mingw-w64-ucrt-x86_64-openssl
+            mingw-w64-ucrt-x86_64-opus
             mingw-w64-ucrt-x86_64-toolchain
+            wget
 
       - name: Install Doxygen
         # GCC compiled doxygen has issues when running graphviz

--- a/docs/building.md
+++ b/docs/building.md
@@ -86,7 +86,7 @@ dependencies=(
   "mingw-w64-ucrt-x86_64-boost"  # Optional
   "mingw-w64-ucrt-x86_64-cmake"
   "mingw-w64-ucrt-x86_64-cppwinrt"
-  "mingw-w64-ucrt-x86_64-curl"
+  "mingw-w64-ucrt-x86_64-curl-winssl"
   "mingw-w64-ucrt-x86_64-graphviz"  # Optional, for docs
   "mingw-w64-ucrt-x86_64-miniupnpc"
   "mingw-w64-ucrt-x86_64-nlohmann-json"

--- a/src/httpcommon.cpp
+++ b/src/httpcommon.cpp
@@ -196,7 +196,12 @@ namespace http {
   bool
   download_file(const std::string &url, const std::string &file) {
     CURL *curl = curl_easy_init();
-    if (!curl) {
+    if (curl) {
+      // sonar complains about weak ssl and tls versions
+      // ideally, the setopts should go after the early returns; however sonar cannot detect the fix
+      curl_easy_setopt(curl, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_2);
+    }
+    else {
       BOOST_LOG(error) << "Couldn't create CURL instance";
       return false;
     }
@@ -214,17 +219,16 @@ namespace http {
       curl_easy_cleanup(curl);
       return false;
     }
+
     curl_easy_setopt(curl, CURLOPT_URL, url.c_str());
     curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, fwrite);
     curl_easy_setopt(curl, CURLOPT_WRITEDATA, fp);
-    curl_easy_setopt(curl, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_2);
-#ifdef _WIN32
-    curl_easy_setopt(curl, CURLOPT_SSL_OPTIONS, CURLSSLOPT_NATIVE_CA);
-#endif
+
     CURLcode result = curl_easy_perform(curl);
     if (result != CURLE_OK) {
       BOOST_LOG(error) << "Couldn't download ["sv << url << ", code:" << result << ']';
     }
+
     curl_easy_cleanup(curl);
     fclose(fp);
     return result == CURLE_OK;

